src/main/java/org/codehaus/groovy/control/customizers/SecureASTCustomizer.java

```diff
@@ -462,120 +662,200 @@ public void addExpressionCheckers(ExpressionChecker... checkers) {
         }
-        this.constantTypesWhiteList = constantTypesWhiteList;
+        this.allowedConstantTypes = allowedConstantTypes;
+    }
+
+    /**
+     * Legacy alias for {@link #setAllowedConstantTypes(List)}
+     */
+    public void setConstantTypesWhiteList(final List<String> allowedConstantTypes) {
+        setAllowedConstantTypes(allowedConstantTypes);
     }
 
     /**
      * An alternative way of setting constant types.
      *
-     * @param constantTypesWhiteList a list of classes.
+     * @param allowedConstantTypes a list of classes.
      */
-    public void setConstantTypesClassesWhiteList(final List<Class> constantTypesWhiteList) {
+    public void setAllowedConstantTypesClasses(final List<Class> allowedConstantTypes) {
         List<String> values = new LinkedList<>();
-        for (Class aClass : constantTypesWhiteList) {
+        for (Class aClass : allowedConstantTypes) {
             values.add(aClass.getName());
         }
         setConstantTypesWhiteList(values);
     }
 
+    /**
+     * Legacy alias for {@link #setAllowedConstantTypesClasses(List)}
+     */
+    public void setConstantTypesClassesWhiteList(final List<Class> allowedConstantTypes) {
+        setAllowedConstantTypesClasses(allowedConstantTypes);
+    }
+
     /**
      * An alternative way of setting constant types.
      *
-     * @param constantTypesBlackList a list of classes.
+     * @param disallowedConstantTypes a list of classes.
      */
-    public void setConstantTypesClassesBlackList(final List<Class> constantTypesBlackList) {
+    public void setDisallowedConstantTypesClasses(final List<Class> disallowedConstantTypes) {
         List<String> values = new LinkedList<>();
-        for (Class aClass : constantTypesBlackList) {
+        for (Class aClass : disallowedConstantTypes) {
             values.add(aClass.getName());
         }
         setConstantTypesBlackList(values);
     }
 
+    /**
+     * Legacy alias for {@link #setDisallowedConstantTypesClasses(List)}
+     */
+    public void setConstantTypesClassesBlackList(final List<Class> disallowedConstantTypes) {
+        setDisallowedConstantTypesClasses(disallowedConstantTypes);
+    }
+
+    public List<String> getDisallowedReceivers() {
+        return disallowedReceivers;
+    }
+
+    /**
+     * Legacy alias for {@link #getDisallowedReceivers()}
+     */
     public List<String> getReceiversBlackList() {
-        return receiversBlackList;
+        return getDisallowedReceivers();
     }
 
     /**
      * Sets the list of classes which deny method calls.
      *
      * Please note that since Groovy is a dynamic language, and
-     * this class performs a static type check, it will be reletively
-     * simple to bypass any blacklist unless the receivers blacklist contains, at
+     * this class performs a static type check, it will be relatively
+     * simple to bypass any disallowed list unless the receivers disallowed list contains, at
```

Comment:
both here and below, I think this was supposed to be 'disallowed receivers list' instead of 'receiver[']s disallowed list'?